src/Controller/CRUDController.php

```diff
@@ -88,7 +88,7 @@ public function setContainer(ContainerInterface $container = null): void
      *
      * @deprecated since version 3.27, to be removed in 4.0. Use Sonata\AdminBundle\Controller\CRUDController::renderWithExtraParams() instead.
      */
-    public function render($view, array $parameters = [], Response $response = null)
+    public function render(string $view, array $parameters = [], ?Response $response = null): Response
```

Comment:
This method should be removed anyway

Reply:
That's right, but I wanted to make the removal more atomic to have a shorter review process